src/appengine/private/components/jobs/jobs.html

```diff
@@ -164,64 +183,96 @@ <h2>Jobs</h2>
     <div class="section">
       <div class="title">Existing jobs</div>
       <div class="body">
-        <paper-input class="search-box" label="Enter search string here" value="{{jobSearchString::input}}">
+        <paper-input class="search-box" label="Enter search string here" type="search" value="{{params.q}}" on-keyup="searchButtonTapped">
```

Comment:
Sure! There is another event which we can use i.e. [onchange](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event). It was mentioned in the note of on-input that if we want to fire a action on an event such as ENTER key pressed, we can use change. Here: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event

Reply:
Not sure if change will be a direct replacement for input, as that only triggers when the value is "committed". Since we want to handle when the "enter" key is pressed, it seem to me the best way to do this is to have two handlers: one for `input` events which just fires the timeout, and one for the keyup event which just tests for "enter" key. This will catch all cases of inputs we care about.

Reply:
yes, that would be best option. Thanks :)

Reply:
done